src/framework/global/api/iapiengine.h

```diff
@@ -36,6 +36,8 @@ class IApiEngine
     virtual QJSValue newQObject(QObject* o) = 0;
     virtual QJSValue newObject() = 0;
     virtual QJSValue newArray(size_t length = 0) = 0;
+
+    virtual void registryMetaObject(const QString& name, const QMetaObject* meta) = 0;
```

Comment:
Should be named `registerMetaObject`. (register = verb, registry = noun 🙂)

Reply:
fixed